LocalLang/Evaluator.lean

```diff
@@ -1,10 +1,64 @@
 import LocalLang.AST
-import Mathlib.Control.Traversable.Basic
+import Std.Data.HashMap.Basic
+import Init.Data.Repr
+
+abbrev Vars := Std.HashMap String ℕ
```

Comment:
I think we have to use the same Env that we defined elsewhere, which also has functions.